docs/program-setup/bare_metal/bare_metal.md

```diff
@@ -18,5 +18,3 @@ The Mbed OS bare metal profile is a compact profile of Mbed OS without an RTOS.
 | Storage | Ecosystem libraries | Available |
 | Mbed TLS | Not available | Available |
 | Mbed PSA | Not available | Available |
-
-To begin using the Mbed OS bare metal profile from Mbed OS 2, please follow our [instructions](../tutorials/migrating-to-mbed-os-5.html).
```

Comment:
We need this line as we have a page to do this. However, it should be slightly modified as follows:
`To begin using the Mbed OS bare metal profile from Mbed 2, please follow our [instructions](../tutorials/xxx.html).`

Reply:
We're redoing the bare metal content in another PR, so I'm going to ignore this comment - the whole page changed in the other PR